config/default.yml

```diff
@@ -106,6 +106,8 @@ AllCops:
   # the `--enable-pending-cops` command-line option.
   # When `NewCops` is `disable`, pending cops are disabled in bulk. Can be overridden by
   # the `--disable-pending-cops` command-line option.
+  # You can also set this to a version number per-department, e.g. `NewCops: 1.1` will
+  # enable all cops that were added in version 1.1 and earlier of the relevant plugin.
```

Comment:
I believe "extension" is a more appropriate term.

Reply:
I believe that extensions and departments are orthogonal. An extension can add cops to a department that exists in the core RuboCop.
Making this as flexible as allowing to configure it per-extension **and** per department is an overkill. `AllCops/NewCops` for RuboCop core and optional separate per-extension is just fine.

WDYT:
```yaml
AllCops:
  NewCops: 1.15
```
equivalent to:
```yaml
AllCops:
  NewCops:
    rubocop: 1.15
```
For extensions:
```yaml
AllCops:
  NewCops:
    rubocop: 1.15
    rubocop-rails: 2.5
```

Reply:
Setting a version number, is ambiguous.
Should a version of RuboCop core or an extension that introduced a certain cop to the existing department be used to decide if the cop is affirmed or not?

I believe some kind of warning should be issued in case of such an ambiguity, e.g.:
```
`Style/NewCops` is set to `1.15`. However, both `rubocop` and `rubocop-nitpicks` provide cops to the `Style` department, and their versions do not align.
```

Reply:
I think a problem will be we don’t know what extension adds what cop but I agree per extension makes more sense to me than per dept.